projects/discussion-ui/src/lib/components/discuss-start/discuss-start.component.html

```diff
@@ -16,17 +16,19 @@
             category is mandatory</small>
         </div>
 
-        <div class="mb-16" [ngClass]="{ 'is-invalid': showError('subTitle') }">
+        <div class="mb-16" >
+          <label class="start-form-label">Discussion topic*</label>
           <div class="start-form-field">
-            <label class="start-form-label" for="discussTopic">Discussion topic*</label>
-            <input class="start-form-field-control py-8" placeholder="Type here (minimum 8 characters)"
-              formControlName="question" #question maxlength="1000" [required]="true" id="discussTopic" />
+            <input class="start-form-field-control py-8" minlength="8" rows="8" placeholder="Type here (minimum 8 characters)"
+              formControlName="question" #question maxlength="1000" />
```

Comment:
We have a formcontrol for the input and we can use to set validators. can you please use it instead of adding minLength in html